README.md

```diff
@@ -78,3 +78,8 @@ $ dbt docs generate
 $ dbt-coverage compute doc --cov-report after.json --cov-fail-compare before.json  # Fail if the current coverage is lower than coverage in before.json
 $ dbt-coverage compare after.json before.json  # Generate a detailed coverage delta report
 ```
+
+
+## Related packaged
```

Comment:
@mrshu, I mentioned the related package at the very bottom of the README because I found the other information much more important. If you think it belongs somewhere else, let me know! 😊

Reply:
@sweco sure, I wouldn't have done it differently